tensorflow/python/ops/nn_grad.py

```diff
@@ -395,6 +395,13 @@ def _AvgPoolGrad(op, grad):
       data_format=op.get_attr("data_format"))
 
 
+@ops.RegisterGradient("AvgPoolGrad")
+def _AvgPoolGradGrad(op, grad):
+  print(op)
```

Comment:
@vrv For gradient of gradient of avg_pool, the inputs are input: `gradients/AvgPool_grad/Shape` and `gradients/Fill` which looks wrong. There's no `x` in the input. Is this wrong?

```
In [2]: x = tf.constant(np.random.RandomState(1).rand(1, 4, 4, 1).astype(np.float32), name="x")

In [3]: y = tf.nn.avg_pool(x, ksize=[1, 2, 2, 1], strides=[1, 2, 2, 1], padding='VALID')

In [4]: dy = tf.gradients(y, x)

In [5]: ddy = tf.gradients(dy, x)
name: "gradients/AvgPool_grad/AvgPoolGrad"
op: "AvgPoolGrad"
input: "gradients/AvgPool_grad/Shape"
input: "gradients/Fill"
attr {
  key: "T"
  value {
    type: DT_FLOAT
  }
}
attr {
  key: "data_format"
  value {
    s: "NHWC"
  }
}
attr {
  key: "ksize"
  value {
    list {
      i: 1
      i: 2
      i: 2
      i: 1
    }
  }
}
attr {
  key: "padding"
  value {
    s: "VALID"
  }
}
attr {
  key: "strides"
  value {
    list {
      i: 1
      i: 2
      i: 2
      i: 1
    }
  }
}
```